types/protocols/index.d.ts

```diff
@@ -1,14 +1,16 @@
-// Type definitions for protocols 1.4
+// Type definitions for protocols 2.0
 // Project: https://github.com/IonicaBizau/protocols
-// Definitions by: Florian Keller <https://github.com/ffflorian>
+// Definitions by: Florian Imdahl <https://github.com/ffflorian>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+/// <reference types="node" />
```

Comment:
this is a module (it has CJS export), so this should be an import for `URL` symbol drectly (e.g. `import { URL } from ...`